### ISSUE TITLE
How to run the tests

### ISSUE THREAD

**Issue by dotlambda** [alpernebbi/git-annex-adapter#4]
I'm trying to run the tests with `python -m unittest`, but I'm getting the following error:
```
======================================================================
FAIL: test_process_git_status (tests.test_process.TestProcessOnEmptyAnnex)
Process should be able to read whole output
----------------------------------------------------------------------
Traceback (most recent call last):
  File /tmp/nix-build-python3.6-git-annex-adapter-0.2.0.drv-0/git-annex-adapter-v0.2.0-src/tests/test_process.py, line 154, in test_process_git_status
    + 'and use git add to track)',
AssertionError: Lists differ: ['On [16 chars]'', 'No commits yet', '', 'nothing to commit ([42 chars]ck)'] != ['On [16 chars]'', 'Initial commit', '', 'nothing to commit ([42 chars]ck)']

First differing element 2:
'No commits yet'
'Initial commit'

  ['On branch master',
   '',
-  'No commits yet',
+  'Initial commit',
   '',
   'nothing to commit (create/copy files and use git add to track)']

----------------------------------------------------------------------
```
Could you please document somewhere how to run the tests? Maybe the cause is that the tests are run in a different order?

**Comment by alpernebbi** [comment]
That's the correct way to run the tests, and what you're getting is a test failure (not an error). Turns out git has changed the `git status` message slightly (`Initial commit` was changed to `No commits yet` for branches without any commits).

**Comment by alpernebbi** [comment]
I've fixed the test on 9f64c4b, (also fixed some debug chatter in another commit) and pushed it.

If you still think I should, I could put a testing primer in the readme file.

**Comment by dotlambda** [comment]
Thanks a lot.
No, I don't think that will be necessary since googling `python unittest` will quickly bring up the documentation: https://docs.python.org/3.5/library/unittest.html#command-line-interface.
